package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Starter kit for apollo server using webpack and typescript",
   "main": "dist/main.js",
   "scripts": {
+    "prebuild" : "npm install",
```

Comment:
I used to do the same, but then removed it,
You will want to start the server most of the times without running install.
So please remove this and we will merge only the README.md